pkg/databackup/constants.go

```diff
@@ -1,7 +1,8 @@
 package databackup
 
 const (
-	FINALIZER        = "fluid-databackup-controller-finalizer"
-	BACPUP_PATH_POD  = "/alluxio_backups"
-	DATABACKUP_CHART = "fluid-databackup"
+	FINALIZER               = "fluid-databackup-controller-finalizer"
+	ALLUXIO_BACPUP_PATH_POD = "/alluxio_backups"
+	GOOSEFS_BACPUP_PATH_POD = "/goosefs_backups"
+	DATABACKUP_CHART        = "fluid-databackup"
```

Comment:
CamelCase ? [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
I think this is a folder name, we can follow the naming habits of folders. : )

Reply:
I mean the name of the variable, not the value

Reply:
ok, clear, lgtm